generic/tohil.c

```diff
@@ -855,7 +882,7 @@ tohil_swap_subinterp(Tcl_Interp *interp)
 //  that it's the parent, so as not to delete it if our
 //  interpreter gets deleted.
 //
-static void
+static PyThreadState *
 tohil_setup_subinterp(Tcl_Interp *interp, enum SubinterpType subtype)
```

Comment:
Could you add a note to the comment for this function about returning the thread state and the non-obvious implicit thread-state-swap that `Py_NewInterpreter()` performs?

Reply:
Done.  Commit 5621b36e6e663db0d6a8490c595e86a4600826ca